behaviour/graql/language/delete.feature

```diff
@@ -549,35 +756,68 @@ Feature: Graql Delete Query
         $x isa person;
         $y isa person;
       """
-
-    Then get answers of graql query
+    Then the integrity is validated
+    When get answers of graql query
       """
       match $r isa friendship; get;
       """
     Then answer size is: 0
 
 
-  Scenario: delete a role player with too-specific (downcasting) role throws
-    Given graql define
-      """
-      define
-      special-friendship sub friendship,
-        relates special-friend as friend;
-      """
-    Given the integrity is validated
-
-    When graql insert
+  Scenario: when the last role player is disassociated from a relation instance, the relation instance gets deleted
```

Comment:
Add this behaviour as an issue to discuss in your "higher priority graql behaviour issues" please, i recall haikal wanted to revisit it at some point :)

Reply:
Is it an issue?

Reply:
I'll bring it up in tomorrow's update